components/common/Header.js

```diff
@@ -1,91 +1,99 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'next/router';
-import Link from 'next/link';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 
 import Clickable from 'components/common/Clickable';
-import ArticleLink from 'components/articles/ArticleLink';
+import Link from 'components/common/Link';
+import LocaleContext from 'components/common/LocaleContext';
 
 import { actions, selectors } from 'redux/ducks/auth';
 import text from 'constants/dictionary';
-import routes from 'constants/routing';
+import { Router, NAVBAR_ROUTES, ROUTES_NAMES } from 'routes';
+import { LOCALES, LANGS } from 'constants';
 
 const mapStateToProps = state => ({ user: selectors.getUser(state) });
-
 const mapDispatchToProps = { signOut: actions.signOut };
 
-/* TODO: get values for langs and links from dictionary ? */
-// const langs = [
-//   { id: 'be', value: 'беларуская' },
-//   { id: 'ru', value: 'русский' },
-//   { id: 'en', value: 'english' },
-// ];
+const checkActive = (path, name) => path.split('/').slice(-1)[0] === name;
```

Comment:
Is it possible to connect routing to the redux and to store current path/route in the store?

Reply:
Hmm, I thinks so, but what pros we get?

Reply:
Perhaps, you won't need to parse path...

Reply:
But there are several cons:
* connect to get URL
* selector to get url
* we should handle every Link & page update and sync it with Redux

I think now it's redundant

Reply:
I believe, it could looks clearer and could be done with less efforts, perhaps, using already existed functionality in next/routing/redux... But probably for this moment it's really redundant